set.mm

```diff
@@ -630953,6 +630953,1026 @@ standardize vectors to a length (norm) of one, but such definitions require
+      WXFUWNUVFUUPWWXSWWXEUYSUVFUUPWWXDUWNWWUSXXUGWVVBUUPVUGWWUBXXVPXXXGTZRHHHH
+      HHHHHHHHKUUPWWXPWWXBWVXSUVFUUPWWXOWWXAWVXTUVFUUPWWXNWWWTUYAUVFUUPWWXMWWWS
+      UYCUVFUUPWWXLWWWRUVTUVFUUPWWXKWWWQUYHUVFUUPWWXJWWWPUYKUVFUUPWWXIWWWOUWDUV
+      FUUPWWXHWWWNUWHUVFUUPWWXGWWWMUYOUVFUUPWWXFWWWLUWNUVFUUPWWXEWWWKUYSUVFUUPV
+      UCUYFWWVOXXUFUUVDUUPUWNWWUSWVVBXXXHTZRHHHHHHHHHHHHKUUPWWXBWWWHWVXSUVFUUPW
+      WXAWWWGWVXTUVFUUPWWWTWWWFUYAUVFUUPWWWSWWWEUYCUVFUUPWWWRWWWDUVTUVFUUPWWWQW
+      WWCUYHUVFUUPWWWPWWWBUYKUVFUUPWWWOWWWAUWDUVFUUPWWWNWWVTUWHUVFUUPWWWMWWVSUY
+      OUVFUUPWWWLWWVRUWNUVFUUPWWWKWWVQUYSUVFUUPWWWJWWVPVUCUVFUUPUYFWWVOUUVDXXXI
+      VSHHHHHHHHHHHHHKUUPWWWHWWVMWVXSUVFUUPWWWGWWVLWVXTUVFUUPWWWFWWVKUYAUVFUUPW
+      WWEWWVJUYCUVFUUPWWWDWWVIUVTUVFUUPWWWCWWVHUYHUVFUUPWWWBWWVGUYKUVFUUPWWWAWW
+      VFUWDUVFUUPWWVTWWVEUWHUVFUUPWWVSWWVDUYOUVFUUPWWVRWWVCUWNUVFUUPWWVQWWVBUYS
+      UVFUUPWWVPWWVAVUCUVFUUPUWNWWUSUYFWVVBXXXHUUVDRHHHHHHHHHHHHHKUUPWWVMWWUQWV
+      XSUVFUUPWWVLWWUPWVXTUVFUUPWWVKWWUOUYAUVFUUPWWVJWWUNUYCUVFUUPWWVIWWUMUVTUV
+      FUUPWWVHWWULUYHUVFUUPWWVGWWUKUYKUVFUUPWWVFWWUJUWDUVFUUPWWVEWWUIUWHUVFUUPW
+      WVDWWUHUYOUVFUUPWWVCWWUGUWNUVFUUPWWVBWWUFUYSUVFUUPWWVAWWUEVUCUVFUUPWWUTWW
+      UDUWNUVFUUPVUGWWUBUYFXXVPXXXGUUVDRHHHHHHHHHHHHHHKUUPWWUQWVYTWVXSUVFUUPWWU
+      PWVYSWVXTUVFUUPWWUOWVYRUYAUVFUUPWWUNWVYQUYCUVFUUPWWUMWVYPUVTUVFUUPWWULWVY
+      OUYHUVFUUPWWUKWVYNUYKUVFUUPWWUJWVYMUWDUVFUUPWWUIWVYLUWHUVFUUPWWUHWVYKUYOU
+      VFUUPWWUGWVYJUWNUVFUUPWWUFWVYIUYSUVFUUPWWUEWVYHVUCUVFUUPWWUDWVYGUWNUVFUUP
+      WWUCWVYFVUGUVFUUPWWUBWVYEUYFUVFUUPUVLVUIVWLVXCVSMHHHHHHHHHHHHHHHKUUPWVYTW
+      VYCWVXSUVFUUPWVYSWVXQWVXTUVFUUPWVYRWVXPUYAUVFUUPWVYQWVXOUYCUVFUUPWVYPWVXL
+      UVTUVFUUPWVYOWVXIUYHUVFUUPWVYNWVXHUYKUVFUUPWVYMWVXDUWDUVFUUPWVYLWVXCUWHUV
+      FUUPWVYKWVWRUYOUVFUUPWVYJWVWQUWNUVFUUPWVYIWVWKUYSUVFUUPWVYHWVWEVUCUVFUUPW
+      VYGVULUWNUVFUUPWVYFVUKVUGUVFUUPVUIUVLUYFVXCVWLUUVDRHHHHHHHHHHHHHHHKUUPWVX
+      SWVXTWVXQXXUCXXVNUUPUYAWVXPXXVOUUPUYCWVXOXXVRUUPUVTWVXLWVUSUUPUYHWVXIXXUD
+      UUPUYKWVXHXXWLUUPUWDWVXDWVUTUUPUWHWVXCWVVAUUPUYOWVWRXXVSUUPUWNWVWQWVVBUUP
+      UYSWVWKXXXEUUPVUCWVWEXXUFUUPUWNVULWVVBUUPVUGVUKXXVPUUPVUIVUJVXCUUPUVLUYFV
+      WLUUVDTTTZTZTZTTZTTZTTZTZTTTZROUUPWVYAVWEWVXQUVFUUPUVKUUQXWYSVYHXWYPUUPXW
+      YRIUUMMKUUPWVXQXXXQUUNKUUPWVXMWVXPUYAUVFUUPUYCUVTWVXLXXVRWVUSXXXPRHOUUPWV
+      XJWVXMUYAUVFUUPUYDUYHWVXIUUPUYCUVTXXVRWVUSTXXUDXXXORHOUUPWVXFWVXJUYAUVFUU
+      PWVXEWVXIUYIUVFUUPUYKUWDWVXDXXWLWVUTXXXNRHHOUUPWVXAWVXFUYAUVFUUPWVWTWVXEU
+      YIUVFUUPWVWSWVXDUYLUVFUUPUWHUYOWVWRWVVAXXVSXXXMRHHHOUUPWVWOWVXAUYAUVFUUPW
+      VWNWVWTUYIUVFUUPWVWMWVWSUYLUVFUUPWVWLWVWRUYPUVFUUPUWNUYSWVWKWVVBXXXEXXXLR
+      HHHHOUUPWVWIWVWOUYAUVFUUPWVWHWVWNUYIUVFUUPWVWGWVWMUYLUVFUUPWVWFWVWLUYPUVF
+      UUPUYTVUCWVWEUUPUWNUYSWVVBXXXETZXXUFXXXKRHHHHOUUPVUPWVWIUYAUVFUUPVUOWVWHU
+      YIUVFUUPVUNWVWGUYLUVFUUPVUMWVWFUYPUVFUUPVUDUWNVULUUPUYTVUCXXXRXXUFTWVVBXX
+      XJRHHHHOUUOYS $.
+
```

Comment:
This proof is really extremly long. Maybe a way to shorten it is to provide theorems like ` \( A \+ B \) ^ 3 = ... ` and to use them in the proof.

Reply:
Such theorems already exist and they are used in the proof: [binom3](https://us.metamath.org/mpeuni/binom3.html), [cu3addd](https://us.metamath.org/mpeuni/cu3addd.html).